appcenter-link-scripts/src/ios/PodFile.js

```diff
@@ -23,11 +23,7 @@ Podfile.prototype.eraseOldLines = function () {
 Podfile.prototype.addPodLine = function (pod, podspec, version) {
```

Comment:
Because parameter `podspec` is no longer used, let's remove it from `addPodLine` function.